src/nvim/screen.c

```diff
@@ -7643,7 +7638,7 @@ void screen_resize(int width, int height)
   if (curwin->w_buffer == NULL)
     return;
 
-  ++busy;
+  busy = true;
```

Comment:
Really not sure about this one, does it pass the tests ?

Reply:
busy can only be zero or one, but renaming it `recursive` would make this more explicit.